<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(90, 90, 90);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="6" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="StatusLabel">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Focus</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QToolButton" name="SettingsTool">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 85, 0);
border-radius: 8px;
padding: 5px;
color: white;</string>
            </property>
            <property name="text">
             <string>Settings</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="PomodoroPage">
            <property name="styleSheet">
             <string notr="true">font: 75 44pt;
color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Pomodoro</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="TimeRemaining">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Time remaining</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" rowspan="4">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="CalendarTool">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 85, 0);
border-radius: 8px;
padding: 5px;
color: white;</string>
            </property>
            <property name="text">
             <string>Calendar</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="StartButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(225, 112, 0);
border-radius: 8px;
padding: 5px;
color: white;</string>
            </property>
            <property name="text">
             <string>START</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SettingsPage">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="SettingsGrid">
          <item row="2" column="2">
           <widget class="QPlainTextEdit" name="ShortEdit"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Long break time:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QPushButton" name="BackToTimerSettings">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(225, 112, 0);
border-radius: 8px;
padding: 5px;
color: white;</string>
            </property>
            <property name="text">
             <string>Back to timer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="2">
           <widget class="QPlainTextEdit" name="LongEdit"/>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QPlainTextEdit" name="FocusTimeEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Focus time:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QPushButton" name="SoundButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(225, 112, 0);
border-radius: 8px;
padding: 5px;
color: white;</string>
            </property>
            <property name="text">
             <string>Sound on/off</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Short break time:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLabel" name="SettingsLabel">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 44pt;
</string>
            </property>
            <property name="text">
             <string>Settings</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CalendarPage">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="1">
         <layout class="QGridLayout" name="CalendarGrid">
          <item row="0" column="3">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLabel" name="CalendarLabel">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 44pt;
</string>
            </property>
            <property name="text">
             <string>Calendar</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QPushButton" name="BackToTimerCalendar">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(225, 112, 0);
border-radius: 8px;
padding: 5px;
color: white;</string>
            </property>
            <property name="text">
             <string>Back to timer</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QCalendarWidget" name="Calendar"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
